smartmin/views.py

```diff
@@ -606,8 +606,8 @@ def get_context_data(self, **kwargs):
         context['link_fields'] = self.link_fields
 
         # our search term if any
-        if 'search' in self.request.REQUEST:
-            context['search'] = self.request.REQUEST['search']
+        if 'search' in self.request.GET:
```

Comment:
As far as I can tell, 'search', '_order' and '_format' are always querystring params so I think it's ok to just use `GET`